src/cms/utils/account_activation_utils.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib import messages
+from django.contrib.auth.tokens import default_token_generator
+from django.core.mail import send_mail, BadHeaderError
+from django.template.loader import render_to_string
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+from django.utils.translation import ugettext as _
+
+from backend.settings import BASE_URL
+
+
+def send_activation_link(request, user_instance):
+    """Sends activation link to user with inactive account
+
+    Making use of ~django.core.mail.send_mail to prevent malicious header injections
+    by raising an ~django.core.mail.BadHeaderError exception
+    if a newline character is detected in subject, recipient_list, etc.
+
+    :param request: The current http request
+    :type request: ~django.http.HttpRequest
+
+    :param user_instance: the users profile to send the email
+    :type user_instance: ~cms.models.users.user_profile.UserProfile
```

Comment:
```suggestion
    :param user_instance: The user to whom the e-mail is to be sent
    :type user_instance: ~django.contrib.auth.models.User
```
I think this is the user model and not the user profile model, right?